numpy/linalg/linalg.py

```diff
@@ -1336,46 +1345,75 @@ def svd(a, full_matrices=True, compute_uv=True):
 
     .. versionadded:: 1.8.0
 
-    Broadcasting rules apply, see the `numpy.linalg` documentation for
-    details.
+    The decomposition is performed using LAPACK routine ``_gesdd``.
+
+    SVD is usually described for the factorization of a 2D matrix :math:`A`.
+    The higher-dimensional case will be discussed below. In the 2D case, SVD is
+    written as :math:`A = U S V^H``, where :math:`A=` ``a``, :math:`U=` ``u``,
```

Comment:
B^H`` ? Too many backticks?

Reply:
Maybe just put the whole bit in latex, ":math:`A=a` ... :math:`B^H=vh`"?

Reply:
Or ``:math:`A = \mathtt{a}` ``, if that's supported

Reply:
Or ``:math:`A = \mathtt{a}` ``, if that's supported

Reply:
I've tried `\mathtt` and it works but the result is not very intuitive. It seems to be common in numpy docstrings to write variable names in italics, with single back ticks, while `\mathtt` does the opposite in Latex. Single back ticks looks more consistent.